tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -435,12 +436,8 @@ const templates = {
       sourceMaps: true,
     };
   },
-  justConfig: stripIndents`
-    import { preset, task } from '@fluentui/scripts-tasks';
-
-    preset();
-
-    task('build', 'build:react-components').cached?.();`,
+  // why not inline template ? this is needed to stop TS parsing static imports and evaluating them in nx dep graph tree as true dependency - https://github.com/nrwl/nx/issues/8938
+  justConfig: getTemplate(joinPathFragments(__dirname, 'files/just-config.ts__tmpl__'), {}),
```

Comment:
why not just use `generateFiles` and apply all template changes?

Reply:
we dont use "templates" stored in separate files in this generator.

having inline templates is usually preferred as it supports better DX and type-checking if one needs dynamic template creation.

this case doesn't need anything dynamic though. so I implemented an "inline" generateFiles for sake of consistency and less refactoring needed. 

to wrap this up, the main culprit for all these workarounds is typescript compiler that's causing those source parsing issues that force us to do these things.